cmd/monaco/main.go

```diff
@@ -351,10 +353,22 @@ func deleteConfigs(apis map[string]api.Api, environments map[string]environment.
 		for name, environment := range environments {
 			util.Log.Info("Deleting %d configs for environment %s...", len(configs), name)
 
+			apiToken, err := environment.GetToken()
+			if err != nil {
+				return err
+			}
+			client := rest.NewDynatraceClient(environment.GetEnvironmentUrl(), apiToken)
```

Comment:
Is there a reason to create one client per environment for execute and another for delete? 

Maybe we could create the client per env once somewhere early in the execution and reuse them - e.g. keep a map of env->client that can be accessed in both execute and delete?

Reply:
Good idea, I already thought of that. However, I wanted to keep it as simple as possible for now. Later, we can do a lot of improvements like parallel execution, caching of the initial GET request results, ... :smile: